src/hotspot/cpu/aarch64/stubGenerator_aarch64.cpp

```diff
@@ -141,7 +141,8 @@ class StubGenerator: public StubCodeGenerator {
   //     [ return_from_Java     ] <--- sp
   //     [ argument word n      ]
   //      ...
-  // -27 [ argument word 1      ]
+  // -29 [ argument word 1      ]
+  // -28 [ saved Floating-point Control Register ]
```

Comment:
Why the 1-word gap? Alignment?

Reply:
Yes, exactly. AArch64 stack is 16-aligned, and this is enforced by hardware.